tbx/project_styleguide/templates/patterns/pages/work/work_page.html

```diff
@@ -95,7 +95,7 @@ <h1 class="heading heading--one">{{ page.title }}</h1>
 
         <ul class="streamfield__related-posts">
             {% for post in page.related_works %}
-                {% include "patterns/molecules/listing/listing--image.html" with title=post.title client=post.client name=post.author.name job_title=post.author.role link=post.url date=post.date reading_time=post.read_time tags=post.related_services listing_image=post.listing_image listing_image=post.listing_image tags_screenreader_text="Filter work posts by category" %}
```

Comment:
Similarly this page has changed completely so it would be easiest to make any changes once the front-end code is merged.

Reply:
as above, this is now merged.